doc/articles/controls/GLCanvasElement.md

```diff
@@ -0,0 +1,196 @@
+---
+uid: Uno.Controls.GLCanvasElement
+---
+
+## GLCanvasElement
+
+> [!IMPORTANT]
+> This functionality is only available on Skia Desktop (`net8.0-desktop`) targets that are running with hardware acceleration. This is also not available on MacOS.
```

Comment:
Adding the TargetFramework here feels confusing. It should still work on old templates that are not using Uno.Sdk, I think?

Reply:
Old templates that are using class lib may actually be problematic, it won't be trivial to get it working there. It can work for the too old templates that used to use shared projects though, but I think no one is using them now. We can keep it netX.0-desktop